### ISSUE TITLE
Missing metadata.rb

### ISSUE THREAD

**Issue by mlcooper** [poise/poise#11]
It appears that metadata.rb is missing for the latest release of poise

**Comment by rhass-r7** [comment]
It appears poise is a gem now. 

https://rubygems.org/gems/poise

**Comment by martinb3** [comment]
It might be useful to have something in the README.md that explains how to use this gem to write resource and provider classes. I think most people understood how to use the cookbook (`include_recipe`), but I'm not sure everyone will know how to use the gem-ified version (`chef_gem`?).

**Comment by coderanger** [comment]
Yep, this is now managed via Halite. The Halite docs have information on how to do the conversion, or you can get a converted copy from supermarket.chef.io.

**Comment by martinb3** [comment]
Thanks Noah :)

**Comment by dvinograd** [comment]
There's plenty of cookbooks still referencing poise cookbook as a dependency in their metadata.rb. For example, elasticsearch cookbook. It took me a while to figure out how to get a copy of poise cookbook 2.0.1, for example, to satisfy elasticsearch cookbook's dependency. First I copied this repository and checked out v2.0.1 but metadata.rb was still missing. So I found that using `knife cookbook site download poise 2.0.1` fetches a copy from supermarket.chef.io. I'm sure someone new to Chef will get completely lost if they don't know where to look.
